README.md

```diff
@@ -35,19 +41,14 @@ few days. The important part is submitting your change.
 
 ## Local Development Environment
 
-The Chef Documentation website is built using [Hugo](https://gohugo.io/) and
-[NPM](https://www.npmjs.com/). You will need Hugo 0.61 or higher installed and
-running to build and view our documentation properly.
-
-To install Hugo:
-
-- On macOS run: `brew install hugo`
-- On Windows run: `choco install hugo`
+The Chef Documentation website is built using [Hugo](https://gohugo.io/),
```

Comment:
The Chef Documentation website is built using:

- [Hugo](https://gohugo.io/) 0.61 or higher
- [Node](https://www.nodejs.com) 10.0.0 or higher
- [NPM](https://www.npmjs.com/) 5.6.0 or higher
- [Go](https://golang.org/dl/) 1.12 or higher